drydock/values.yaml

```diff
@@ -0,0 +1,48 @@
+
+# Copyright 2017 The Openstack-Helm Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file provides defaults for drydock
+
+replicas:
+  drydock: 1
+
+images:
+  drydock: quay.io/attcomdev/drydock:1.0.1
+  dep_check: docker.io/kolla/ubuntu-source-kubernetes-entrypoint:4.0.0
+  pull_policy: "IfNotPresent"
+
+network:
+  drydock:
+    port: 9000
+    node_port:
+      enabled: false
```

Comment:
I think this will need to be enabled for now as newly provisioned nodes will need to hit the Drydock API to pull in some metadata. We can override it in the armada file if that is the better solution.